scalac-scoverage-plugin/src/main/scala/scoverage/report/CoberturaXmlWriter.scala

```diff
@@ -39,7 +39,7 @@ class CoberturaXmlWriter(sourceDirectories: Seq[File], outputDir: File) extends
   }
 
   def klass(klass: MeasuredClass): Node = {
-    <class name={klass.name}
+    <class name={klass.fullClassName}
```

Comment:
I've just checked it (`a.b.c` - package names, `Foo.Boo.Moo` - nested objects):
- Maven Java project with Cobertura
```xml
<class name="a.b.c.Foo$Boo$Moo" filename="a/b/c/Foo.java" line-rate="1.0" branch-rate="1.0" complexity="1.0">
    <methods>
        <method name="hello" signature="()Ljava/lang/String;" line-rate="1.0" branch-rate="1.0" complexity="0">
            ...
        </method>
        ...
</class>
```
Maven Scala project with scalac-scoverage-plugin 1.1.1
```xml
<class name="Moo" filename="a/b/c/Foo.scala" line-rate="1.00" branch-rate="1.00" complexity="0">
    <methods>
        <method name="a.b.c/Moo/hello" signature="()V" line-rate="1.00" branch-rate="1.00">
            ...
        </method>
        ...
</class>
```
Maven Scala project with scalac-scoverage-plugin 1.2.0 with your PR applied (current master):
```xml
<class name="a.b.c.Foo.Boo" filename="a/b/c/Foo.scala" line-rate="0.00" branch-rate="0.00" complexity="0">
    <methods>
        <method name="a.b.c/Moo/hello" signature="()V" line-rate="1.00" branch-rate="1.00">
            ...
        </method>
        ...
</class>
```
So now is better, but not ideal. There should be dollar signs in class name. You did not change method name, but it could be fixed with class name fix.